packages/slate/src/interfaces/editor.ts

```diff
@@ -757,9 +757,13 @@ export const Editor = {
           `)
         }
 
-        const path = getDirtyPaths(editor).pop()!
-        const entry = Editor.node(editor, path)
-        editor.normalizeNode(entry)
+        const dirtyPath = getDirtyPaths(editor).pop()!
+
+        // If the node doesn't exist in the tree, it does not need to be normalized.
+        if (Node.has(editor, dirtyPath)) {
+          const entry = Editor.node(editor, dirtyPath)
+          editor.normalizeNode(entry)
+        }
         m++
```

Comment:
```suggestion
        const dirtyPath = getDirtyPaths(editor).pop()!

        // If the node doesn't exist in the tree, it does not need to be normalized.
        if (!Node.has(editor, dirtyPath)) continue 
        
        const entry = Editor.node(editor, dirtyPath)
        editor.normalizeNode(entry)
        m++
```

Reply:
Thanks for the input! I made the change, but I noticed a potential conflict.

With the continue line, the rest of the code is skipped. Which would also mean line 766 (`m++`) will never happen, which could potentially create a scenario where an error is not thrown when it should be due an incorrect number of iterations being kept.

Because of this I have moved that counter to before the continue statement. If this is not needed, I will move it back to the end of the loop.

Reply:
I think you're right.  If you want to move it back to your original formulation, that actually may be a bit cleaner.  I can then merge it in.  Thanks!

Reply:
Thanks! I went ahead and changed it back and I will resolve this thread